packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -195,6 +195,7 @@ export const Animation = React.forwardRef<HTMLDivElement, AnimationProps>((props
       className={!isChildrenFunction ? cx(animationClasses, className, (child as any)?.props?.className) : ''}
     >
       {isChildrenFunction ? (
+        // @ts-ignore - @types/react-transition-group doesn't actually include this API, nor is it documented
```

Comment:
I suspect this is incorrect usage of RTG, there has never been an API that uses `({ state })` in the types in any major version, nor is it documented on the RTG docsite. However, ts-ignore is better than changing northstar IMO